packages/react/src/index.js

```diff
@@ -7,43 +7,43 @@ sizeMe.noPlaceholders = true;
-export EmptyTable from './components/Table/EmptyTable/EmptyTable';
-export Table from './components/Table';
-export TableHead from './components/Table/TableHead/TableHead';
-export TableBody from './components/Table/TableBody/TableBody';
-export TableSaveViewModal from './components/Table/TableSaveViewModal/TableSaveViewModal';
-export TableManageViewsModal from './components/Table/TableManageViewsModal/TableManageViewsModal';
-export TableSkeletonWithHeaders from './components/Table/TableSkeletonWithHeaders/TableSkeletonWithHeaders';
-export TableToolbar from './components/Table/TableToolbar/TableToolbar';
-export WizardModal from './components/WizardModal';
-export WizardInline from './components/WizardInline/WizardInline';
-export StatefulWizardInline from './components/WizardInline/StatefulWizardInline';
-export StatefulTable from './components/Table/StatefulTable';
-export TileCatalog from './components/TileCatalog/TileCatalog';
-export StatefulTileCatalog from './components/TileCatalog/StatefulTileCatalog';
-export CatalogContent from './components/TileCatalog/CatalogContent';
-export SimplePagination from './components/SimplePagination/SimplePagination';
-export ProgressIndicator from './components/ProgressIndicator/ProgressIndicator';
-export ComposedStructuredList from './components/ComposedStructuredList/ComposedStructuredList';
-export ResourceList from './components/ResourceList/ResourceList';
-export FileDrop from './components/FileDrop/FileDrop';
-export PageTitleBar from './components/PageTitleBar/PageTitleBar';
-export HierarchyList from './components/List/HierarchyList';
-export ListContent from './components/List/ListContent/ListContent';
-export VirtualListContent from './components/List/VirtualListContent/VirtualListContent';
-export BarChartCard from './components/BarChartCard/BarChartCard';
-export TileCatalogNew from './components/TileCatalogNew/TileCatalogNew';
-export TimePickerSpinner from './components/TimePickerSpinner/TimePickerSpinner';
-export DateTimePicker from './components/DateTimePicker/DateTimePicker';
-export DateTimePickerV2 from './components/DateTimePicker/DateTimePickerV2';
-export TableViewDropdown from './components/Table/TableViewDropdown/TableViewDropdown';
-export IconDropdown from './components/IconDropdown/IconDropdown';
-export EmptyState from './components/EmptyState/EmptyState';
-export ImageGalleryModal from './components/ImageGalleryModal/ImageGalleryModal';
+export { default as AddCard } from './components/AddCard';
+export { default as Button } from './components/Button';
+export { default as Breadcrumb, BreadcrumbItem, BreadcrumbSkeleton } from './components/Breadcrumb';
+export { default as ComposedModal } from './components/ComposedModal';
+export { default as TableHead } from './components/Table/TableHead/TableHead';
+export { default as EmptyTable } from './components/Table/EmptyTable/EmptyTable';
+export { default as Table } from './components/Table';
```

Comment:
this re-ordering was accidental (I was trying to track down all the import cycle errors) and is fixed in a later commit (it also jacked up the publicApi snapshot output and needed to be reverted)